flatflow/scheduler/internal/algorithm/regression.h

```diff
@@ -0,0 +1,109 @@
+// Copyright 2024 The FlatFlow Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef FLATFLOW_SCHEDULER_INTERNAL_ALGORITHM_REGRESSION_H_
+#define FLATFLOW_SCHEDULER_INTERNAL_ALGORITHM_REGRESSION_H_
+
+#include <omp.h>
```

Comment:
It seems there is no use of OpenMP in this file, so this ``include`` statement should be omitted.